.configure

```diff
@@ -5,7 +5,7 @@
   "files_to_copy": [
```

Comment:
(We definitively need to make our `configure` tool support a `files_to_remove` entry there at some point, to be able to nicely deprecate old locations…)

Reply:
This is a super good idea